### DISCUSSION TITLE
Programm: Aktionen für Geräte nicht auswählbar

### DISCUSSION THREAD

**Discussion by JoWrusch** [OpenCCU/OpenCCU#2856] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

![image](https://github.com/user-attachments/assets/942f9303-a4f4-4ebc-a2aa-63d6347f066d)
Für Geräte ist keine Aktion mehr auswählbar, obwohl die Geräte jeweils manuell über das Menu steuerbar sind.
Es erscheint lediglich die Auswahl 'Kommunikation war gestört: Ja' oder 'Kommunikation war gestört: Nein'.

### Describe the behavior you expected

Auswahl von Einschalten und Ausschalten

### Steps to reproduce the issue

1. Öffnen des Programms
2. Auswahl eines Gerätes
3. Auswahl der Aktion
...

### What is the version this bug report is based on?

3.77.7.20240826

### Which base platform are you running?

rpi3 (RaspberryPi3, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

HM-MOD-RPI-PCB

### Anything in the logs that might be useful for us?

```txt

```

### Additional information

Bitte angeben welchen Bereich des Logs eingereicht werden sollte

**Comment by jens-maus** [comment, marked as answer]
Wie man im Screenshot sieht wurde hier ja auch Kanal 0 (siehe :0) ausgewählt. Dieser hat natürlich die Einschalten/Ausschalten aktionen nicht. Es müsste also ein anderer Kanal dieses Gerätes gewählt werden der dann die entsprechenden Aktionen hat.

**Comment by JoWrusch** [comment]
Vielen Dank für die schnelle Antwort.
Manchmal sind es die kleinen Dinge, die man nicht sieht....

**Comment by JoWrusch** [comment]
Danke !!!!

**Comment by jens-maus** [comment]
> Vielen Dank für die schnelle Antwort. Manchmal sind es die kleinen Dinge, die man nicht sieht....

Das ist trotzdem kein Bugreport wert, sondern sollte zuerst in einem Forum diskutiert werden um eigene Fehler (wie hier zu sehen) auszuschließen.
